src/components/dialog/styles/dialog.css

```diff
@@ -0,0 +1,49 @@
+@import '../../../styles/animations.css';
+@import '../../../styles/variables.css';
+
+.main {
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	background: #FFF;
+	width: 450px;
+	min-height: 250px;
+	overflow: hidden;
+	max-width: calc(100% - 30px);
+	font: var(--base-font);
+	box-shadow: var(--subtle-shadow);
+	border-radius: var(--rounded-corners);
+}
+
+.underlay {
+	width: 100%;
+	position: fixed;
+	top: 0;
+	left: 0;
+	height: 100%;
+}
+
+[data-underlay="true"] .underlay {
+	background: var(--dojo-shade-gray);
```

Comment:
Why are you using these colour vars directly rather than abstracting them away? ie. if we changed `gray` to be a different colour the naming would then be wrong.